news/2022-09-27-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,153 @@
+
+If you would like to know more about the changes posted, or for a more detailed breakdown on the some of the in-development changes, head on over to the [osu!taiko rework document](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit). This is a working document, and will periodically be updated with the latest developments from any developers who request their changes to be added to it, via the osu!taiko pp committee.
+
+## An insight into the base of the new rework
+
+The last set of changes made to osu!taiko's performance points & star rating algorithm aimed to chart several skills at once. This not only made it difficult to collect meaningful and targeted feedback relevant to one specific skill, but also led to increased complexity which halted continuous development efforts. 
+
+As such, we have opted to take a more modular approach by introducing new changes incrementally, in the hopes of being able to fix any reported issues one step at a time. Moving forward, osu!taiko star rating will be receiving continuous updates, potentially placing star rating and performance points in a state of flux, especially over the coming few months. 
+
+During this time, we will be working on other difficulty components (both small and large-scale) in preparation for future releases. It is our hope that this will continue to help the algorithm reflect the latest trends and expectation in difficulty calculation, but we also apologise in any advance for any disruption this may cause as we begin working things out.
+
+In the star rating part of this rework, we have essentially completely reworked the stamina and colour skills, introducing new concepts and the infrastructure required to support them.
+
+Performance points have seen significant changes to mod multipliers and a broad change to how accuracy is calculated. Please note however that these changes are *experimental* and did not receive as much scrutiny as the rest of the rework. Consider leaving feedback if you find anything that feels especially unusual.
+
+Without further ado, let's get into the nitty-gritty of these changes.
+
+## Star rating changes
+
+### Readdressing stamina
+
+#### Issues within the old system
+
+The old system assumed a 2-finger kddk full alternating playstyle (i.e. one in which each subsequent hit is always hit by the other hand regardless of colour). This was implemented as two skills — one for the left hand, and one for the right hand. Instead of using note duration for the split skills, it used the duration between two notes that were pressed with the same hand, a term that we describe as note-pair duration.
+
+In addition to the above, tailored and highly particular nerfs targeted at specific patterns were implemented using code described best as a "cheese detector", which is something that we have since opted to avoid. 
```

Comment:
```suggestion
In addition to the above, tailored and highly particular nerfs targeted at specific patterns were implemented using code described best as a "cheese detector", which is something that we have since opted to do away with.
```